src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -23,47 +23,52 @@
 
 (defn- address-input
   [input-value input-focused?]
-  (fn []
-    (let [current-screen-id        (rf/sub [:navigation/current-screen-id])
-          scanned-address          (rf/sub [:wallet/scanned-address])
-          send-address             (rf/sub [:wallet/wallet-send-to-address])
-          recipient                (rf/sub [:wallet/wallet-send-recipient])
-          recipient-plain-address? (= send-address recipient)
-          valid-ens-or-address?    (rf/sub [:wallet/valid-ens-or-address?])
-          chain-id                 (rf/sub [:chain-id])
-          contacts                 (rf/sub [:contacts/active])]
-      [quo/address-input
-       {:on-focus              #(reset! input-focused? true)
-        :on-blur               #(reset! input-focused? false)
-        :on-scan               (fn []
-                                 (rn/dismiss-keyboard!)
-                                 (rf/dispatch [:wallet/clean-scanned-address])
-                                 (rf/dispatch [:open-modal :screen/wallet.scan-address]))
-        :ens-regex             constants/regx-ens
-        :scanned-value         (or (when recipient-plain-address? send-address) scanned-address)
-        :address-regex         constants/regx-multichain-address
-        :on-detect-address     #(when (or (= current-screen-id :screen/wallet.select-address)
-                                          (= current-screen-id :screen/wallet.scan-address))
-                                  ; ^ this check is to prevent effect being triggered when screen is
-                                  ; loaded but not being shown to the user (deep in the navigation
-                                  ; stack) and avoid undesired behaviors
-                                  (debounce/debounce-and-dispatch
-                                   [:wallet/validate-address %]
-                                   300))
-        :on-detect-ens         (fn [text cb]
-                                 (when (or (= current-screen-id :screen/wallet.select-address)
-                                           (= current-screen-id :screen/wallet.scan-address))
-                                   ; ^ this check is to prevent effect being triggered when screen
-                                   ; is loaded but not being shown to the user (deep in the
-                                   ; navigation stack) and avoid undesired behaviors
-                                   (debounce/debounce-and-dispatch
-                                    [:wallet/find-ens text contacts chain-id cb]
-                                    300)))
-        :on-change-text        (fn [text]
-                                 (when (empty? text)
-                                   (rf/dispatch [:wallet/clean-local-suggestions]))
-                                 (reset! input-value text))
-        :valid-ens-or-address? valid-ens-or-address?}])))
+  (let [set-scanned-value-fn (atom nil)]
+    (fn []
```

Comment:
`input-focused?` is not being received here, we should receive both parameters if we are going to react to those values